### ISSUE TITLE
License clarification

### ISSUE THREAD

**Issue by collimarco** [zinclabs/zinc#196]
Hello, thanks for this great project.

I see that this software is released under the "Apache-2.0 license", is that correct?

It seems that the license text is not recognized by GitHub: in the sidebar you can see "View license" instead of "Apache-2.0 license" (like in many other projects).

Are there any specific words or clauses that were changed from the original license?

Otherwise if the text is correct you should probably contact GitHub and report the issue with your repository.

**Comment by prabhatsharma** [comment]
It is Apache 2.0 . LICENSE file has never changed since its initial commit and it used to be recognized by github. Somehow it stopped being recognized. It is something wrong at github's end.

TLDR - It is standard Apache 2.0 with no modifications.
